app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,22 @@
  */
 .cancel {
   text-align: center;
+}
+
+body {
+ background-image: image_url("");
+ background-size: cover;
+ background-repeat: no-repeat;
+ background-position: center top;
+ background-position: 0px 45px;
```

Comment:
Property `background-position` already defined on line 26<br>Line should be indented 2 spaces, but was indented 1 spaces<br>`0px` should be written without units as `0`